Button/index.html

```diff
@@ -0,0 +1,265 @@
+            padding-right: 0px;
+          "
+        >
+          <iframe
+            src="https://carbon.now.sh/embed?bg=rgba%28249%2C237%2C212%2C1%29&t=twilight&wt=none&l=htmlmixed&width=680&ds=false&dsyoff=0px&dsblur=4px&wc=true&wa=false&pv=0px&ph=0px&ln=false&fl=1&fm=IBM+Plex+Mono&fs=18px&lh=160%25&si=false&es=2x&wm=false&code=%253Cbutton%2520class%253D%2522btn%2522%2520disabled%253ELARGE%2520BUTTON%253C%252Fbutton%253E%253Cbutton%2520class%253D%2522btn%2520btn-primary%2520btn-sm%2522%2520disabled%253ESMALL%2520BUTTON%253C%252Fbutton%253E"
+            style="
+              width: 680px;
+              height: 160px;
+              border: 0;
+              transform: scale(1);
+              overflow: hidden;
+            "
+            sandbox="allow-scripts allow-same-origin"
+          >
+          </iframe>
+        </div>
+        <hr />
+        <h4>Outline buttons</h4>
+        <p>
+          In need of a button, but not the hefty background colors they bring?
+        </p>
+        <div class="container flex-container">
+          <button class="btn btn-outline-primary">Primary</button>
+          <button class="btn btn-outline-secondary">Secondary</button>
+          <button class="btn btn-outline-success">Success</button>
+          <button class="btn btn-outline-warning">Warning</button>
+          <button class="btn btn-outline-info">Info</button>
+        </div>
+        <div
+          class="container iframe"
+          style="
+            height: 300px;
+            background-color: transparent;
+            padding-left: 0px;
+            padding-right: 0px;
+          "
+        >
+          <iframe
+            src="https://carbon.now.sh/embed?bg=rgba%28249%2C237%2C212%2C1%29&t=twilight&wt=none&l=htmlmixed&width=680&ds=false&dsyoff=0px&dsblur=4px&wc=true&wa=false&pv=0px&ph=0px&ln=false&fl=1&fm=IBM+Plex+Mono&fs=18px&lh=160%25&si=false&es=2x&wm=false&code=%253Cbutton%2520class%253D%2522btn%2520btn-outline-primary%2522%253EPrimary%253C%252Fbutton%253E%250A%253Cbutton%2520class%253D%2522btn%2520btn-outline-secondary%2522%253ESecondary%253C%252Fbutton%253E%2520%253Cbutton%2520class%253D%2522btn%2520btn-outline-success%2522%253ESuccess%253C%252Fbutton%253E%253Cbutton%2520class%253D%2522btn%2520btn-outline-warning%2522%253EWarning%253C%252Fbutton%253E%253Cbutton%2520class%253D%2522btn%2520btn-outline-info%2522%253EInfo%253C%252Fbutton%253E"
+            style="
+              width: 680px;
+              height: 201px;
+              border: 0;
+              transform: scale(1);
+              overflow: hidden;
+            "
+            sandbox="allow-scripts allow-same-origin"
+          >
+          </iframe>
+        </div>
+        <hr />
+        <h4>Link Buttons</h4>
+        <p>In need of a button, that is a link ?</p>
+        <div class="container flex-container">
+          <a class="btn btn-link">Primary</a>
+        </div>
+        <div
+          class="container iframe"
+          style="
```

Comment:
We can have a class for these properties.

Reply:
Due to the iframe, I have to manually change every style. Carbon. sh is not working properly even if for embed